frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -127,11 +135,11 @@
 <script>
 import sbp from '@sbp/sbp'
 import { mapGetters } from 'vuex'
+import 'vue-virtual-scroller/dist/vue-virtual-scroller.css'
```

Comment:
It seems this should be removed?

Reply:
Hm, not sure, maybe some of those styles are being used (but in that case, you're right still that it should be removed and vendored). Thanks for noticing it.